tensorboard/components/tensor_widget/health-pill-types.ts

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the 'License');
```

Comment:
`'License'` should be `"License"`. Here and below and other files.

Reply:
Done throughout the files created in this PR.